src/modules/endpoint/Endpoint.js

```diff
@@ -113,16 +109,15 @@ class Endpoint {
         return new StoreResponse(false)
       }
 
-      const formattedURL = format(this.url, urlParams)
-      /*
-       todo: check if there are any paramters left in the url: formattedURL.match(/{(.*)?}/)
-       currently this does not work as unused paramaters are just removed from the url
-       */
+      const formattedURL = this.mapRouterToUrl(router)
+
+      if (formattedURL.includes('undefined')) {
```

Comment:
Do we need to use a string here? `[undefined].includes(undefined)` is true for example.

Reply:
which is okay because we won't ever have an endpoint where the request url is undefined. Do you have a idea how to solve this more elegant anyway?

Reply:
yep, but you can just write `if (formattedURL.includes(undefined)) {`